_data/backup.yml

```diff
@@ -236,3 +236,12 @@ websites:
     tfa: Yes
     software: Yes
     doc: http://www.zetta.net/about/blog/how-set-zetta-two-factor-authentication/
+
+  - name: Zoolz
+    url: https://intelli.zoolz.com/
```

Comment:
I'm not sure we should use intelli subdomain. I think it's better to use www or just apex and let them redirect wherever they'll decide to...